demisto_sdk/commands/changelog/changelog.py

```diff
@@ -0,0 +1,151 @@
+        )
+
+    """ VALIDATE """
+
+    def validate(self) -> bool:
+        """
+        ...
+        """
+        if self.is_release():
+            if not self.is_changelog_folder_empty():
+                logger.error("Something msg")
+                return False
+            if not self.is_changelog_changed():
+                logger.error("Something msg")
+                return False
+        else:
+            if self.is_changelog_changed():
+                logger.error("Something msg")
+                return False
+            if not self.is_changelog_yml_exist():
+                logger.error("Something msg")
+                return False
+            if not self.validate_changelog_yml():
+                logger.error("Something msg")
+                return False
+        return True
+
+    """ INIT """
+
+    def init(self) -> None:
+        """
+        Creates a new changelog file for the current PR
+        """
+        if self.is_release():
+            logger.error(
+                "This PR is for release, please use in changelog release command"
+            )
+        initial_changelog = {
+            "description": "enter description about this PR",
+            "type": "<fix|feature|breaking>",
+        }
+        try:
+            new_opj = ChangelogObject(**initial_changelog)
+        except ValidationError as e:
+            logger.error(e.json())
+
+        with Path(f"{git_path()}/.changelog/{self.pr_number}.yml").open("w") as f:
+            yaml.dump(new_opj.dict(), f)
+
+        logger.info("Something msg")
+
+    """ RELEASE """
+
+    def release(self) -> None:
+        if not self.is_release():
+            logger.error("Something msg")
+            return
+        changelogs = self.get_all_changelogs()
+        self.edit_changelog_file(changelogs)
+        self.cleaning_changelogs_folder()
+
+    """ HELPER FUNCTIONS """
+
+    def is_changelog_changed(self) -> bool:
+        ...
+
+    def is_changelog_folder_empty(self) -> bool:
+        return any(CHANGELOG_FOLDER.iterdir())
+
+    def is_changelog_yml_exist(self) -> bool:
+        return Path(f"{git_path()}/{self.pr_number}.yml").is_file()
+
+    def validate_changelog_yml(self) -> bool:
+        data = get_yaml(Path(f"{git_path()}/.changelog/{self.pr_number}.yml"))
+        try:
+            ChangelogObject(**data)
+        except ValidationError as e:
+            logger.error(e.json())
+            return False
+        return True
+
+    def get_all_changelogs(self) -> List[ChangelogObject]:
+        changelogs: List[ChangelogObject] = []
+        for path in CHANGELOG_FOLDER.iterdir():
+            changelog_data = get_yaml(path)
+            try:
+                changelogs.append(ChangelogObject(**changelog_data))
+            except ValidationError as e:
+                logger.error(f"{path}: {e.json()}")
+
+        return changelogs
+
+    def edit_changelog_file(self, changelogs: List[ChangelogObject]) -> None:
+        fixes = tuple(
```

Comment:
would be prettier to use [itertools.groupby](https://docs.python.org/3/library/itertools.html#itertools.groupby) or [more_itertools.bucket](https://more-itertools.readthedocs.io/en/stable/api.html#more_itertools.bucket) instead of three `tuple(filter(`s